notebooks/Analyze_Model_Outputs.ipynb

```diff
@@ -95,8 +95,7 @@
        "  <thead>\n",
```

Comment:
<p><code>sentence</code>  references <code>TokenSpan</code>,  maybe remove the reference or briefly mention what it means?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Yeah, that reference comes out of the blue in this version. I've removed the reference to `TokenSpan` here, since there is a detailed explanation of token-based spans a few cells later in the same notebook.